pkg/cmd/cli/bug/bug.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2018 the Heptio Ark contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bug
+
+import (
+	"bytes"
+	"fmt"
+	"net/url"
+	"os/exec"
+	"runtime"
+	"text/template"
+
+	"github.com/heptio/ark/pkg/buildinfo"
+	"github.com/heptio/ark/pkg/cmd"
+	"github.com/spf13/cobra"
+)
+
+const (
+	linuxBrowserLaunchCmd = "xdg-open"
+	issueURL              = "https://github.com/heptio/ark/issues/new"
+	issueTemplate         = `
```

Comment:
Is there any way to avoid having the template hard-coded here? We already have it in `.github/ISSUE_TEMPLATE/bug_report.md`, but I realize that's not (at the moment) necessarily easy to fill in programatically.

Reply:
there's also https://github.com/GeertJohan/go.rice, but it seems a bit weird to me.

Reply:
Embedding the template in go does make a bit of copy-pasta. Could `.github/ISSUE_TEMPLATE/bug_report.md` be generated from a go template with empty variables somewhere in the build process, with the same template being loaded/downloaded + populated in `ark bug`?

Reply:
I'm not sure how well that would work - as it stands, I think GitHub uses that file directly, without any pre-processors. I think we'd like to keep that integration working.

I do suppose it's possible to use a command that updates it and we commit the result, similar to how we do the cli-reference docs.

@jhamilton1 Would having a command that dumps contents of a Go variable into `.github/ISSUE_TEMPLATES/bug_report.md` be workable for your usage? It would mean updates to the text would happen inside a Go file, but the format should still be Markdown.

Reply:
> I do suppose it's possible to use a command that updates it and we commit the result, similar to how we do the cli-reference docs.

That's what I failed to articulate :-)

Reply:
I think this would need:

- hack/update-generated-issue-template.sh
- hack/verify-generated-issue-template.sh
- a raw issue template
  - the issue "template template"
- a tiny program to render this template file (similar to https://github.com/heptio/ark/blob/master/docs/generate/ark.go)
- a generated issue template (`.github/ISSUE_TEMPLATE/bug_report.md`) made with the program above
- the `ark bug` command would need to download the raw template and populate it's fields to open a new issue on Github
- maybe more?